pandas/plotting/_core.py

```diff
@@ -1277,6 +1277,9 @@ def hist(self, by=None, bins=10, **kwargs):
         ----------
         by : str or sequence, optional
             Column in the DataFrame to group by.
+
+            .. versionadded:: 1.3.0
```

Comment:
yeah, I was doubting about this. We do accept `by` right now but don't do anything about that. Now we do start supporting `by` to make plots for groups, so i put `added` here instead of `changed`.

I think then i will change to `versionchanged`.

oh, i just realised this should be `1.4`, i will change!